nullaway/src/test/java/com/uber/nullaway/tools/SerializationTestHelper.java

```diff
@@ -59,12 +59,12 @@ public SerializationTestHelper(Path outputDir) {
 
   @SafeVarargs
   public final SerializationTestHelper<T> setExpectedOutputs(T... outputs) {
-    this.expectedOutputs = Arrays.asList(outputs);
+    this.expectedOutputs = ImmutableList.copyOf(Arrays.asList(outputs));
```

Comment:
You can directly pass the `outputs` array to `copyOf(...)`. There is a version of that method that takes an array and this avoids the double list construction.

Reply:
Oh, great. Looks much better.

aaf7d4743fb6236be9dafd9e658522b893c498d6